lib/Sema/SemaOverload.cpp

```diff
@@ -12405,6 +12405,10 @@ static FunctionDecl *getBestCandidateForHIP(Sema &S,
     return Cnt0 < Cnt1;
   });
 
+  // Validate number of arguments.  Add 5 for launch configuration
+  if ( (It->Function->param_size()+5u) != Args.size())
```

Comment:
Instead of `+5u` here, it would be better to move IgnoreCnt into file scope and use it here.